examples/dataset/object_detection_2d/object_detection_2d/commons.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import defaultdict
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Union
+
+import pandas as pd
+
+from kolena.workflow.annotation import BoundingBox
+from kolena.workflow.annotation import LabeledBoundingBox
+from kolena.workflow.annotation import ScoredLabeledBoundingBox
+
+BUCKET = "kolena-public-datasets"
```

Comment:
Once we have finalized the csv files for the v1 examples, we should move them to the new bucket: `s3://kolena-public-examples`.  Please see the other examples for reference.